examples/example_scheduler.go

```diff
@@ -56,6 +56,7 @@ var (
 	taskCount           = flag.String("task-count", "5", "Total task count to run.")
 	mesosAuthPrincipal  = flag.String("mesos_authentication_principal", "", "Mesos authentication principal.")
 	mesosAuthSecretFile = flag.String("mesos_authentication_secret_file", "", "Mesos authentication secret file.")
+	role                = flag.String("role", "*", "Role to login framework with")
```

Comment:
A framework doesn't **login** with a role, it gets assigned a role. Here, however, you're only assigning the role to resources, not the framework.